src/webots/nodes/WbRobot.cpp

```diff
@@ -54,6 +54,7 @@
 #include <QtCore/QStringList>
 #include <QtCore/QTimer>
 
+#include <iostream>
```

Comment:
I assume this is for debugging?

Reply:
It was, forgot to delete, I discovered `QTextStream(stdout) <<` is much better for most of the cases.